acoustid/api/v2/__init__.py

```
@@ -67,17 +67,17 @@
 MAX_RESULTS_PER_FINGERPRINT_QUERY = 10
 
 
-def iter_args_suffixes(args, *prefixes):
-    results = set()
-    for name in args.keys():
+def iter_args_suffixes(args, *prefixes) -> Iterable[str]:
```

Ruff (0.8.2)

Missing type annotation for function argument `args`

(ANN001)